verify/captcha_server.py

```diff
@@ -0,0 +1,129 @@
+from flask import Flask, render_template, request, redirect, url_for, session, flash
+import requests
+import os # For a secure session key
+
+app = Flask(__name__)
+
+# --- Configuration ---
+# IMPORTANT: Set a strong secret key for Flask sessions!
+# You can generate one using: python -c 'import os; print(os.urandom(16))'
+app.secret_key = b'_5#y2L"F4Q8z\n\xec]/' # Replace with YOUR generated key
```

Comment:
store these key in .env file 

inside file should containsomething like
GCP_PROJECT_ID="kdfjg kldfjhgkldg"
...
here is example of the usage

import os
from dotenv import load_dotenv

load_dotenv()

GCP_PROJECT_ID = os.getenv('GCP_PROJECT_ID') 
SERVICE_ACCOUNT_FILE = os.getenv('SERVICE_ACCOUNT_FILE')
STORAGE_BUCKET_NAME = os.getenv('STORAGE_BUCKET_NAME')